### PR TITLE
ID-3451 [FIX] Fixes an issue where some of the form fields have empty settings

### PR DESCRIPTION
**Pull request by armine-fliplet**

### CODE REVIEW

File: templates/configurations/form.interface.hbs [hunk 1]
```diff
@@ -175,7 +175,7 @@
 
     {{#unless isTypeahead}}
     <div v-if="_componentsWithPersonalization.includes(_componentName)" class="form-group cleafix">
-      {{> defaultValue}}
+      {{defaultValue}}
```
Comment (tonytlwu) [inline templates/configurations/form.interface.hbs:178]: Interesting. Can you explain why this worked? I thought partials are rendered through `{{> }}`? https://handlebarsjs.com/guide/partials.html

Comment (armine-fliplet) [reply templates/configurations/form.interface.hbs:178]: Honestly, I don't know, the error occurred with `>` this for form interface, but it worked fine for typeahead interface.

Comment (tonytlwu) [reply templates/configurations/form.interface.hbs:178]: @armine-fliplet Could you investigate and find out why? We don't want unintended consequences so we should know why it works for one context but not for the other (or why `>` is required in one but not the other).

Comment (armine-fliplet) [reply templates/configurations/form.interface.hbs:178]: @tonytlwu see the ChatGPT answer [here](https://chat.openai.com/share/9d4b5d44-13d5-4725-992a-22f07d99a470)

**Review by tonytlwu — APPROVED** [review]

**Pull request merged**